.github/workflows/ruby.yml

```diff
@@ -0,0 +1,26 @@
+name: Ruby
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+    - name: Build and run Rubocop
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+        bundle exec rubocop --parallel
```

Comment:
Because this is on a fork, you will not see the status of the workflow run here.